src/idownloadmanager.h

```diff
@@ -34,6 +34,16 @@ class QDLLEXPORT IDownloadManager : public QObject
    */
   virtual int startDownloadNexusFile(int modID, int fileID) = 0;
 
+  /**
+   * @brief download a file from www.nexusmods.com/<gameName>.
+   * @param gameName 'short' name of the game the mod is for
+   * @param modID id of the mod for which to download a file
+   * @param fileID id of the file to download
+   * @return an id by which the download will be identified
+   */
+  virtual int startDownloadNexusFileForGame(const QString& gameName, int modID,
```

Comment:
I would use the game ID instead of the game name, I think those are less likely to be changed in the future (if I remember correctly, Nexus has changed the game short names a few times in the past which was kind of annoying for MO2).